docs/src/usage/getting_started.md

```diff
@@ -28,24 +27,8 @@ Similarly, ADRIA can be updated as new releases are made:
 julia> ]up ADRIA
 ```
 
-To setup ADRIA for development, see the [Development Setup](@ref) page.
-
-## Visualizations
-
-The Makie package ecosystem is used for producing plots.
-These are optional, and only necessary if visualizations are desired.
-
-```julia-repl
-julia> ]add GLMakie GeoMakie GraphMakie
-```
-
-## Running ADRIA simulations
-
 Before anything can be done, a `config.toml` file is needed.
-Create a `config.toml` file inside your project directory
-with the following content below (adjusted for your needs).
-
-This `config.toml` file is specific to your computer and project. It **should not be** committed to version control.
+Create a `config.toml` file inside your project directory with the following content below (adjusted for your needs):
```

Comment:
This bit should be updated to indicate that a `config.toml` file is optional, and that the assumed default values are shown below.